src/NuGet.Core/NuGet.Protocol.Core.v3/DownloadResourceV3.cs

```diff
@@ -170,13 +168,16 @@ private DownloadResourceV3(HttpSource client)
                 }
                 catch (IOException ex) when (ex.InnerException is SocketException && i < 2)
                 {
-                    string message = $"Error downloading {identity} from {uri} {ExceptionUtilities.DisplayMessage(ex)}";
-
+                    string message = string.Format(Strings.Log_ErrorDownloading, identity, uri)
+                        + Environment.NewLine
+                        + ExceptionUtilities.DisplayMessage(ex);
                     logger.LogWarning(message);
                 }
                 catch (Exception ex)
                 {
-                    throw new FatalProtocolException(ex);
+                    string message = string.Format(Strings.Log_ErrorDownloading, identity, uri);
+                    logger.LogError(message + Environment.NewLine + ExceptionUtilities.DisplayMessage(ex));
+                    throw new FatalProtocolException(message, ex);
```

Comment:
add an extra blank line above the throw, to make it readbale, same with the logger.LogWarning